scripts/credhub_shell.sh

```diff
@@ -2,26 +2,32 @@
 
 set -eu
 
-SSH_PATH=${SSH_PATH:-"/Users/${USER}/.ssh/id_rsa"}
+SSH_PATH=${SSH_PATH:-"${HOME}/.ssh/id_rsa"}
```

Comment:
```suggestion
SSH_PATH=${SSH_PATH:-"${HOME}/.ssh/id_rsa"}
```

We can remove this and use the key in the SSH agent, this will allow Yubikeys